src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -38,20 +39,26 @@
 {
     use EagerLoadingTrait;
 
+    const ENABLE_MAX_DEPTH = 'enable_max_depth';
+    const NORMALIZATION_CONTEXT = 'normalization_context';
```

Comment:
I don't think its a good idea to introduce does public constants here. They should at least be market with `@internal` (private constants are in PHP 7.1 only). But I'll remove them totally.